### ISSUE TITLE
Bug: .session files stored with ":" character

### ISSUE THREAD

**Issue by Gi7w0rm** [tsale/TeleTracker#13]
Hey there, 
as discussed via Twitter, the fact that .session files are stored containing the  ":" leads to problems when trying to copy files from one place to another. 
Please replace ":" by "_" or something similar, then adjust the tool so that it can use those session files to continue session later.

Cheers
Gi7

**Comment by Gi7w0rm** [comment]
Update: 
Further evaluation has lead me to the conclusion that the .session files created by this program are actually part of the pyrogram library.

The behavior is described here:

https://docs.pyrogram.org/topics/storage-engines

It can potentially be fixed by setting the session storage to "in_memory" by replacing

`  app = pyrogram.Client(bot_token, api_id, api_hash)`
  
 with 
`   app = pyrogram.Client(bot_token, api_id, api_hash, in_memory=True)`
   
   in line 52 of TeleViewer.py .
   
   Currently testing this potential fix.

**Comment by Gi7w0rm** [comment]
Update:
Switching to full "in memory" session storage ruins the file download feature as repeated login attempts lead to 

`Error: Telegram says: [420 FLOOD_WAIT_X] - A wait of 1656 seconds is required (caused by "auth.ImportBotAuthorization")
`

**Comment by Gi7w0rm** [comment]
Solution:
The problem can be solved by replacing current line:

`  app = pyrogram.Client(bot_token, api_id, api_hash)`
  
with 
```
  bot_token_filename = bot_token.replace(":", "_").replace("/", "_")
  app = pyrogram.Client(bot_token_filename, api_id, api_hash)
```

in line 52 of TeleViewer.py 

Currently, the bot token is passed as filename for the session.
My fix sets the session filename to the bottoken, replacing potential ":" and "/" with an underscore.

This way, session files do not get written with ":" characters, which are not allowed on some filesystems.

**Comment by AnaxyJod** [comment]
For me the session files is not even creating, 
And it's showing the error unable to open the database file when I tried to download all the messages in that channel, I have created .env and set my api id and api hash in it 
If u can solve my problem, that would be nice, thnx

**Comment by tsale** [comment]
@Gi7w0rm I believe this is fixed here for Pyrogram version [v2.1.16](https://github.com/KurimuzonAkuma/pyrogram/releases/tag/v2.1.16)

However, the update is not yet on PyPI, therefore, I will implement your solution according to the latest comment. Once the update is out, we can re-evaluate. Thank you! 🙏
